src/other/utils.jl

```diff
@@ -104,10 +104,12 @@ if VERSION >= v"1.4"
                 len = length(x)
                 if nt > 1 && len > basesize
                     tasks = [Threads.@spawn begin
-                                 for i in p
-                                     local $(esc(lidx)) = @inbounds x[i]
-                                     $(esc(lbody))
-                                 end
+                                 let p = p
+                                     for i in p
```

Comment:
I suspect `$p` should be enough, without a `let` (which `spawn` already includes). Are we even sure a change is needed? I don't see how tests could pass if that was the case.

Reply:
I was not sure if it was not needed, and also I was not sure how `$` would be handled in `quote`d code, so I thought using `let` is safest. Do you think it would have any negative impact (for sure it is a safe approach).